go/flags/endtoend/vtcombo.txt

```diff
@@ -421,6 +421,9 @@ Flags:
       --vtgate_grpc_server_name string                                   the server name to use to validate server certificate
       --vttablet_skip_buildinfo_tags string                              comma-separated list of buildinfo tags to skip from merging with --init_tags. each tag is either an exact match or a regular expression of the form '/regexp/'. (default "/.*/")
       --wait_for_backup_interval duration                                (init restore parameter) if this is greater than 0, instead of starting up empty when no backups are found, keep checking at this interval for a backup to appear
+      --warming-reads-percent int                                        Percentage of reads on the primary to forward to replicas. Useful for keeping buffer pools warm (default 0)
+      --warming-reads-pool-size int                                      Size of goroutine pool for warming reads (default 500)
```

Comment:
In Vitess, pools usually mean connection pools. When I read the description I thought it's a waitGroup or something like that. Actually it turns out to be a `channel bool` whose capacity is set by this flag.
The flag name and description are misleading. They need to be changed to reflect the actual usage. It should be something like `--warming-reads-concurrency` and be documented as `Number of concurrent warming reads allowed`.

Reply:
I edited the PR description at the top to list all 3 flags. Once these changes are made, that needs to change again.